pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "pycytominer"
+version = "1.0.0"
+description = "Processing perturbation profiling readouts."
+authors = [
+    "Gregory Way <gregory.way@gmail.com>",
+]
+maintainers = [
+    "Gregory Way <gregory.way@gmail.com>",
+]
+license = "BSD-3-Clause"
+readme = "README.md"
+homepage = "https://pycytominer.readthedocs.io/"
+repository = "https://github.com/cytomining/pycytominer"
+
+[tool.poetry.dependencies]
+python          = ">=3.8"
+numpy           = ">=1.16.5"
+scipy           = ">=1.5"
+pandas          = ">=1.2.0"
+scikit-learn    = ">=0.21.2"
+sqlalchemy      = ">=1.3.6,<2"
+pyarrow         = ">=8.0.0"
+
+fsspec  =   {version = ">=2023.1.0",    extras=["cell_locations"]}
+s3fs    =   {version = ">=0.4.2",       extras=["cell_locations"]}
+boto3   =   {version = ">=1.26.79",     extras=["cell_locations"]}
+fire    =   {version = ">=0.5.0",       extras=["cell_locations"]}
+
+cytominer-database = {version = "0.3.4", extras=["collate"]}
+
+[tool.poetry.group.dev.dependencies]
+pytest = ">=5.0.1"
+pytest-cov = "^4.1.0"
+codecov = ">=2.1.12"
+pre-commit = ">=3.3.2"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.black]
+line-length = 88
+target-version = ['py38']
```

Comment:
Consider aligning this version with what's found in `pre-commit-ci.yml` (Python 3.9). If there's a reason not to do this, consider adding a comment here to describe the version selection.

Reply:
Done